### ISSUE TITLE
confusing terminology active/stopped

### ISSUE THREAD

**Issue by ghistes** [naggie/dstask#119]
Hi,

this is just a minor annoyance, but I (NOT coming from taskwarrior) find the terminology used a little bit confusing.

You enter a task.
Then you "start" it. But the new state is not "started" (as I would have expected) but "active".
Then you "stop" it. Again the new state is not "stopped" but "paused".

This probably comes from taskwarrior and as I said is not a big deal, but e.g. not having a command "list-active", but rather "list-started" would make it self-explanatory what command you would have to use to ("start" obviously) to get a task into this state.

**Comment by naggie** [comment]
I agree it needs to be more consistent. I'll have a think about this and make sure it's better before 1.0.

**Comment by Dieterbe** [comment]
related: #89
